wordpress-mobile/WordPress-iOS
# Add search to language selector

Replaced the generic `SettingsListPickerViewController` with a specific ImmuTable based VC, since adding the search logic would have required to make the generic picker too aware of the language details, and the picker wasn't used anywhere else.

Fixes #4828 

To test:

- Go to a site's settings and change the language
- The language selector should show a search field
- Verify that search works and selecting a new language works as well

![language-search](https://cloud.githubusercontent.com/assets/8739/23953886/26828af8-0996-11e7-99b4-4ea40fa85161.png)

Needs review: @jleandroperez
